src/manifest.ts

```diff
@@ -23,11 +23,13 @@ export class SlackManifest {
         // todo: is there a more idiomatic way of defining this? constant file?
         "major_version": 2,
       },
+
+      "function_runtime": this.getFunctionRuntime(),
       "display_information": {
         background_color: def.backgroundColor,
         name: def.name,
         long_description: def.longDescription,
-        short_description: def.description,
+        description: def.description,
```

Comment:
This was previously broken

Reply:
Yes, this is called `description` under the [`webapp display_information schema`](https://slack-github.com/slack/webapp/blob/master/js/modern/generated/types/api/schemas/app-manifests/v1/display_information/schema.ts)`, so changed the ``short_description`` to  description to prevent invalidation.